docs/appendices/environment_vars.rst

```diff
@@ -0,0 +1,103 @@
+| TOIL_DOCKER_NAME       | The name of the Toil Appliance image you           |
+|                        | wish to use. Generally this is simply ``toil`` but |
+|                        | this option is provided to override this,          |
+|                        | since the image can be built with arbitrary names. |
++------------------------+----------------------------------------------------+
+| TOIL_AWS_ZONE          | The EC2 zone to provision nodes in if using        |
+|                        | Toil's provisioner.                                |
++------------------------+----------------------------------------------------+
+| TOIL_AWS_AMI           | ID of the AMI to use in node provisioning. If in   |
+|                        | doubt, don't set this variable.                    |
++------------------------+----------------------------------------------------+
+| TOIL_AWS_NODE_DEBUG    | Determines whether to preserve nodes that have     |
+|                        | failed health checks. If set to ``True``, nodes    |
+|                        | that fail EC2 health checks won't immediately be   |
+|                        | terminated so they can be examined and the cause   |
+|                        | of failure determined. If any EC2 nodes are left   |
+|                        | behind in this manner, the security group will     |
+|                        | also be left behind by necessity as it cannot be   |
+|                        | deleted until all associated nodes have been       |
+|                        | terminated.                                        |
++------------------------+----------------------------------------------------+
+| TOIL_SLURM_ARGS        | Arguments for sbatch for the slurm batch system.   |
+|                        | Do not pass CPU or memory specifications here.     |
+|                        | Instead, define resource requirements for the job. |
+|                        | There is no default value for this variable.       |
++------------------------+----------------------------------------------------+
+| TOIL_GRIDENGINE_ARGS   | Arguments for qsub for the gridengine batch        |
+|                        | system. Do not pass CPU or memory specifications   |
+|                        | here. Instead, define resource requirements for    |
+|                        | the job. There is no default value for this        |
+|                        | variable.                                          |
++------------------------+----------------------------------------------------+
+| TOIL_GRIDENGINE_PE     | Parallel environment arguments for qsub and for    |
+|                        | the gridengine batch system. There is no default   |
+|                        | value for this variable.                           |
++------------------------+----------------------------------------------------+
+| TOIL_TORQUE_ARGS       | Arguments for qsub for the Torque batch system.    |
```

Comment:
Hello @mr-c. Yes, your contributions to the documentation are welcome. You should be able to edit this PR in github with your new permissions. Let me know if you have any problems.